v3/tutorials/08-ink-workshop/a-getting-started/index.mdx

```diff
@@ -244,7 +244,7 @@ After successfully installing [`substrate-contracts-node`](https://github.com/pa
 you can start a local development chain by running:
 
 ```bash
-substrate-contracts-node --dev --tmp
+substrate-contracts-node --dev
 ```
 
 ![Substrate Smart Contracts Node](/assets/tutorials/ink-workshop/start-canvas-node.png)
```

Comment:
This screenshot also needs to be updated at some point to use only `--dev`, but I figured it was pretty minor

Reply:
Added it to the ToDo here: https://github.com/paritytech/ink/issues/1007.